src/winetricks

```diff
@@ -532,8 +532,11 @@ w_try_regedit32()
 {
     # on windows, doesn't work without cmd /c
     case "$W_PLATFORM" in
-        windows_cmd|wine_cmd) cmdc="cmd /c";;
-        *) unset cmdc ;;
+        windows_cygwin|wine_cmd) cmdc="cmd /c" ;;
+        nix_linux|nix_freebsd|nix_darwin) unset cmdc ;;
+        nix_redox) w_die "Platform '$W_PLATFORM' is not implemented in w_try_regedit32()" ;;
+        windows_windows) w_die "Platform '$W_PLATFORM' is not implemented in w_try_regedit32()" ;;
```

Comment:
Why on earth are you dying here? Windows works just fine as is.

Reply:
@austin987 I don't have way to test this and currently windows is unable to run shell/bash by itself afaik.

Note that `windows_windows` assumes native implementation where `windows_cygwin` and `windows_wsl` are cygwin and wsl